plugins/maciVoting/pages/proposal-list.tsx

```diff
@@ -107,20 +133,16 @@ export default function Proposals() {
                   itemsCount={proposalCount}
                   pageSize={DEFAULT_PAGE_SIZE}
                   state={dataListState}
-                  // onLoadMore={fetchNextPage}
                 >
                   <DataList.Container
                     SkeletonElement={ProposalDataListItemSkeleton}
                     errorState={errorState}
                     emptyFilteredState={emptyFilteredState}
                   >
-                    {proposalCount &&
-                      Array.from(Array(proposalCount)?.keys())
-                        .reverse()
-                        ?.map((proposalIndex) => (
-                          // TODO: update with router agnostic ODS DataListItem
-                          <ProposalCard key={proposalIndex} proposalId={BigInt(proposalIndex)} />
-                        ))}
+                    {proposalIds.map((proposalId) => (
+                      // TODO: update with router agnostic ODS DataListItem
```

Comment:
I am not 100% sure what `// TODO: update with router agnostic ODS DataListItem`. It was in the aragon template, should we remove it? 🤔

Reply:
Yeah no idea either. Mentioned as a todo in https://github.com/privacy-scaling-explorations/maci-aragon-osx-gov-app/issues/59